content/guides/_index.md

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
Is this the right header syntax here? @olizilla or @momack2 or @cwaring or @meiqimichelle -- is this something on which you can advise? Thank you!